manifests/concourse.yml

```diff
@@ -18,7 +18,11 @@ stemcells:
 instance_groups:
 - name: web
   instances: 1
-  vm_type: web
+  # replace with a vm type from your cloud config's "vm_types" `bosh cloud-config`
```

Comment:
This reads funny, mashing the instruction and the shell command into a single sentence.  What about:

```
# replace with a VM type from your BOSH Director's cloud config. Run `bosh cloud-config` and look for `vm_types`
```